rest_framework/relations.py

```diff
@@ -375,7 +375,7 @@ def to_representation(self, iterable):
 
     @property
     def choices(self):
-        queryset = self.child_relation.queryset
+        queryset = self.child_relation.queryset or []
```

Comment:
I'd be more comfortable with this change  if the empty list test was specifically `if queryset is None`, rather than evaluating the boolean value of the queryset.